README.md

```diff
@@ -29,32 +29,40 @@ Read our [Contribution Guidelines](https://github.com/99designs/gqlgen/blob/mast
 ### How do I prevent fetching child objects that might not be used?
 
 When you have nested or recursive schema like this:
+
 ```graphql
 type User {
-    id: ID!
-    name: String!
-    friends: [User!]!
+	id: ID!
```

Comment:
what happened to the indenting here?

Reply:
I went ahead and tried to fix the indentation for a few README snippets. VSCode got a little crazy with the auto formatting.

Let me know if there's anything else you see that needs tweaking.